src/offices/_single.html

```diff
@@ -100,22 +101,30 @@ <h2 class="u-visually-hidden">
             {% if resource.icon %}
             <div class="media_image-container
                         media_image-container__wide-margin">
-                <img class="media_image u-centered-on-mobile"
-                     width="150"
+                <img class="media_image
```

Comment:
So the resource icon is an image file that is uploaded to the office edit page in the WP admin.